injectLatestProps.js

```diff
@@ -12,7 +35,9 @@ exec(
     console.log(`stderr: ${stderr}`);
   }
 );
-exec("cp -r ./react-native-elements/website/docs/props ./src/content/Props"),
+
+execSync(
+  "cp -r ./react-native-elements/website/docs/props ./src/content/Props",
```

Comment:
This is UNIX based path can you use something like this -> https://www.npmjs.com/package/cp-file

Reply:
> This is UNIX based path can you use something like this -> https://www.npmjs.com/package/cp-file

sure